abstractions/ruby/microsoft_kiota_abstractions/lib/microsoft_kiota_abstractions/authentication/base_bearer_token_authentication_provider.rb

```diff
@@ -1,26 +1,22 @@
 # frozen_string_literal: true
 
 require 'concurrent'
+require_relative './authentication_provider'
+require_relative './access_token_provider'
 
 module MicrosoftKiotaAbstractions
   # Provides a base class for implementing AuthenticationProvider for Bearer token scheme
   class BaseBearerTokenAuthenticationProvider
-    include MicrosoftKiotaAbstractions::AuthenticationProvider
     include Concurrent::Async
 
-    AUTHORIZATION_HEADER_KEY = 'Authorization'
-    def authenticate_request(request)
```

Comment:
why was the implementation removed? it should provide a default implementation using the access token provider to avoid duplication